test/helpers/kubectl.go

```diff
@@ -91,7 +91,7 @@ var (
 		"agent.image":                         "cilium-dev",
 		"preflight.image":                     "cilium-dev", // Set again in init to match agent.image!
 		"global.tag":                          "latest",
-		"operator.image":                      "operator",
+		"operator.image.repository":           "operator-generic",
```

Comment:
any case we have two _generic_ here ? https://github.com/cilium/cilium/pull/11837/files#diff-58754dc409d3caca0cbf3bb702b551d3R124